css/default.css

```diff
@@ -556,6 +556,10 @@ table tbody td img {
         max-width: 100%;
         margin: inherit;
     }
+
+    #jumpto {
```

Comment:
@itay1313 FYI (just to point to the more interesting pieces)

Reply:
Good one

Reply:
we can also delete this jumpto.css("display", "flex") add display flex in CSS and not use !important but it was like that anyway in 767 so it's cool

Reply:
Would be awesome if you like to give this another go afterwards, and clean up! Didn’t want to meddle with this so took this route.